### ISSUE TITLE
About teleport

### ISSUE THREAD

**Issue by Pt-Djefferson** [FINDarkside/TLD-Save-Editor#21]
How about safe teleport points - inside buildings or caves or poi? I think I can collect coords of that points.
For now it can be done with Region ComboBox with a lil trick (cause ComboBox not editable):
I have saves in some buildings most important for me, so I load one of that saves, region of that save (Barn for example) now in ComboBox, then I load my current save and change region in ComboBox for Barn, then save, then load in game. Coords in buildings near main door almost always near 0,0,0 (2.799663,-7.201617,0.09310496 for ex. rear door at QuonsetGasStation)
Or some radio buttons can be placed (they can?) at map (zoom???) for teleport only to that points with well known coords.

**Comment by FINDarkside** [comment]
Yeah it's a good idea, but at the moment I don't have much time so I'll only do the bare minimum to keep this editor working. 

More interesting way to get valid locations for all interiors would be to send (opt in) the location and coordinates to my firebase database.

**Comment by ploedman** [comment]
How do you guys handling falling through the map while teleporting?

**Comment by FINDarkside** [comment]
At some point, the game always spawned you on the ground, no matter what your z coordinate was. It was changed at some point and now you'd have to set it manually. In theory it'd be possible to have some kind of height map which would be used to set z coordinate automatically, but that'd be a lot of work. I'm not really working on this editor anymore other than for maintenance reasons.

**Comment by ploedman** [comment]
I tried with minus and plus coordinates for the z axis. But looks like the game does not care, what you put on the z axis. You fall trough the map, and it teleports you to the middle of the map. Anyways never used it before, was just curious if it works.

Thank you for such a great application, wish I could donate to you, using it since the beta!

**Comment by FINDarkside** [comment]
It should work just fine, just tested it. You probably just put in too small z coordinate, or your x and y coordinates might be off the map as well. You can try it by increasing z coordinate when you're already on the ground, you'll spawn mid air.

**Comment by enricodias** [comment]
I just tried the teleport for the first time and couldn't make it work, the character falls through the void. This is related to #14 as well. Changing the z axis manually also didn't work.

I know you don't have much time to deal with this, but I'm just commenting this here in case someone else face the same issue.

**Comment by FINDarkside** [comment]
Changing z axis should work, you can confirm by changing coordinates to some position you know to work. If your z is too low you'll fall through the map and if it's too high you'll fall to your death.
